include/swift/AST/AutoDiff.h

```diff
@@ -422,7 +422,14 @@ class DerivativeFunctionTypeError
   Kind kind;
 
   /// The type and index of a differentiability parameter or result.
-  using TypeAndIndex = std::pair<Type, unsigned>;
+  /// std::pair does not have a trivial copy constructor on FreeBSD <= 14 for
```

Comment:
This comment implies that FreeBSD 15 changed libc++ ABI to support this, but that's not true, it still hasn't happened, and it's not clear if or when it will

Reply:
NB: Upstream libc++ still has the patch to break ABI and support this for FreeBSD >= 14 (not > 14 as this implies), but the vendored copy in FreeBSD used to build /usr/lib/libc++.so.1 is patched to keep the old ABI for all versions.